### PR TITLE
Sort terraform literals in ELB task

### LINKED ISSUE #3578: Terraform: Continuous API ELB subnet changes
`kops version` Version 1.7.0

When running  `kops update cluster --target=terraform --name xxx --out=.` multiple times the `kubernetes.tf` frequently shows changes of the API ELB subnet order. We already tried to checkin the file, but it keeps changing.

```diff
resource "aws_elb" "api-xxx" {
  name = "api-xxx-f-rkbgpe"

  listener = {
    instance_port     = 443
    instance_protocol = "TCP"
    lb_port           = 443
    lb_protocol       = "TCP"
  }

  security_groups = ["${aws_security_group.api-elb-xxx.id}"]
-  subnets         = ["${aws_subnet.us-east-1b-xxx.id}", "${aws_subnet.us-east-1c-xxx.id}", "${aws_subnet.us-east-1a-xxx.id}"]
+  subnets         = ["${aws_subnet.us-east-1a-xxx.id}", "${aws_subnet.us-east-1b-xxx.id}", "${aws_subnet.us-east-1c-xxx.id}"]

  health_check = {
    target              = "TCP:443"
    healthy_threshold   = 2
    unhealthy_threshold = 2
    interval            = 10
    timeout             = 5
  }

  idle_timeout = 300

  tags = {
    KubernetesCluster = "xxx"
    Name              = "api.xxx"
  }
}
```

It looks like https://github.com/kubernetes/kops/pull/1570 didn't fully resolve the ordering issues, but I don't really understand so far, why it isn't working.

### PR DESCRIPTION
**Pull request by justinsb**
Fixes #3578

### CODE REVIEW

File: upup/pkg/fi/cloudup/terraform/literal.go [hunk 1]
```diff
@@ -76,16 +79,39 @@ func sortLiterals(v []*Literal, dedup bool) ([]*Literal, error) {
...
+
 	sort.Sort(byKey(proxies))
 
-	var sorted []*Literal
+	for i := range proxies {
+		v[i] = proxies[i].literal
+	}
+}
+
+func DedupLiterals(v []*Literal) ([]*Literal, error) {
```
Comment (chrislovecnm) [inline upup/pkg/fi/cloudup/terraform/literal.go]: Missing code comment. Sorry to nit pick, but I have been asked to cleanup my docs many times.

Comment (justinsb) [reply upup/pkg/fi/cloudup/terraform/literal.go]: Fixed

**Comment by justinsb** [comment]
Added comment PTAL @chrislovecnm

**Pull request merged**